src/containers/Tools.vue

```diff
@@ -56,13 +72,52 @@ export default {
   }
 
   &-cards {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: flex-start;
+    display: grid;
+    grid-auto-columns: min-content;
+
+    column-gap: 5rem;
+    row-gap: 3rem;
+
+    @media only screen and (max-width: 1300px) {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-around;
+    }
   }
 
   &-card {
-    //  CSS handled in Card component. Go there to adjust margins.
+    &-progress {
+      grid-column: 1/2;
+      grid-row: 1/2;
+    }
+
+    &-advisors {
+      grid-column: 1/2;
+      grid-row: 2/3;
+    }
+
+    &-export {
+      grid-column: 2/3;
+      grid-row: 1/2;
+    }
+
+    &-links {
+      grid-column: 2/3;
+      grid-row: 2/3;
+    }
```

Comment:
I'm not a huge fan of how the styling is coupled to the number of toolcards.

Reply:
I'm not either. I initially tried to do without it, but it just isn't possible to get the design we want for the page AND nicely handle a smaller screen without it. I tried nested flexes and one big flex and it really didn't turn out nicely.